spec/sailfish_iptables_dbus_interface_description.xml

```diff
@@ -0,0 +1,464 @@
+"<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
```

Comment:
Leading quote?

Reply:
Removed in a636c6bfbd986ba333f506dbd593fdc470c13aa7